cfg/Config.mk

```diff
@@ -70,7 +70,7 @@ endif
 ## Also ensure that core memory functions (e.g., memcpy) are included in the build and not name-mangled.
 ## We keep these flags separate from the regular CARGOFLAGS for purposes of easily creating a sysroot directory.
 BUILD_STD_CARGOFLAGS += -Z unstable-options
-BUILD_STD_CARGOFLAGS += -Z build-std=core,alloc
+BUILD_STD_CARGOFLAGS += -Z build-std
```

Comment:
is this no longer needed? Or is this just a change that's left over from experimenting with building std?

If we only need this for "normal" builds of Theseus (without the new std lib) but we cannot use it for builds of the new std lib, then we can change it elsewhere, e.g., as an export override on one of the non-std make targets